docs/tutorial/support.md

```diff
@@ -48,9 +48,9 @@ basis for some older supported lines. All contested decisions around release
 line backports will be resolved by the [Releases Working Group](https://github.com/electron/governance/tree/master/wg-releases) as an agenda item at their weekly meeting the week the backport PR is raised.
 
 ### Currently supported versions
+- 6.x
```

Comment:
nit: let's put `6.x.y`, `5.x.y` etc. for consistency with referenced in the rest of this file.